packages/react-core/src/components/Page/Page.tsx

```diff
@@ -265,12 +266,12 @@ class Page extends React.Component<PageProps, PageState> {
     let nav = null;
     if (horizontalSubnav && isHorizontalSubnavWidthLimited) {
       nav = (
-        <div className={css(styles.pageMainNav, styles.modifiers.limitWidth)}>
-          <div className={css(styles.pageMainBody)}>{horizontalSubnav}</div>
+        <div className={css(styles.pageMainSubnav, styles.modifiers.limitWidth)}>
+          <PageMainBody>{horizontalSubnav}</PageMainBody>
         </div>
       );
     } else if (horizontalSubnav) {
-      nav = <div className={css(styles.pageMainNav)}>{horizontalSubnav}</div>;
+      nav = <div className={css(styles.pageMainSubnav)}>{horizontalSubnav}</div>;
     }
 
     const crumb = breadcrumb ? (
```

Comment:
Just a question but could we use `<PageBreadcrumb>` here? https://github.com/patternfly/patternfly-react/blob/980971bab884928888780067e629538d8902eb47/packages/react-core/src/components/Page/Page.tsx#L278-L292